app/views/statics/donate.html.erb

```diff
@@ -11,7 +11,7 @@
   <li>Donator+ ($20 or more)
 </ul>
 
-<p>We also have <%= link_to "list of users who donated", users_path(badge: "donor") %> already!</p>
+<p>We also have <%= link_to "list of users who donated", users_path(donor: "") %> already!</p>
```

Comment:
This will come out as `We also have list of users who...`. This isn't part of the issue, but if you would add an `a` in there, that would be great. Otherwise, as long as you've tested it and have made sure it works, I approve this.